docs/framework/network-programming/networkinformation.md

```diff
@@ -20,6 +20,6 @@ The <xref:System.Net.NetworkInformation> namespace enables you to gather informa
 ## See also
 
 - [Network Programming Samples](network-programming-samples.md)
-- [Network Information Technology Sample](https://go.microsoft.com/fwlink/?LinkID=179564)
-- [NetStat Tool Technology Sample](https://go.microsoft.com/fwlink/?LinkID=179562)
-- [Ping Client Technology Sample](https://go.microsoft.com/fwlink/?LinkID=179565)
+- [Network Information Technology Sample](https://archive.msdn.microsoft.com/nclsamples/Wiki/View.aspx?title=Network%20Information)
```

Comment:
@davidsh @karelz have these NCL samples been ported somewhere?

Reply:
@mairaw, I think this is the NetStat Tool Technology Sample.

https://github.com/KenAdeniji/WordEngineering/blob/23aa5c2629e6db2618afa6f9fa843a4370da18b9/InformationInTransit/ProcessLogic/NetStatTool.cs

Reply:
The comment in it exactly matches the Remarks appears in the WayBack machine:

![image](https://user-images.githubusercontent.com/31348972/68644665-67643680-051e-11ea-8ca8-a4cae279aea9.png)

Reply:
I've also found [this](https://github.com/Mooliecool/windows/tree/master/sdk/samples/FrameworkSamples/NCL), which contains all the samples.

Reply:
>@davidsh @karelz have these NCL samples been ported somewhere?

I am not aware of any porting efforts for those samples. If they are ported, they should be placed in some GitHub repo.

@mairaw do we have an existing GitHub repo for samples?

Reply:
Yes, we host all our samples at https://github.com/dotnet/samples.

Reply:
Hi all, what should we do in this case? Thanks!

Reply:
@eddynaka, I think @mairaw can suggest where the samples need to be ported in the samples repo and just update the link and any occurrences of it to the samples repo. Not sure if it's good idea to add it to samples browser or not.

Reply:
@davidsh do we have these samples hosted internally somewhere? I'd prefer to grab them from official sources.

Reply:
I've opened #15913 for us to discuss that so we can unblock this PR